components/docked-composer/__examples__/minimized.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console, react/prop-types */
+import React from 'react';
+
+import DockedComposer from '~/components/docked-composer';
+import Accordion from '~/components/accordion/__examples__/base';
+
+class Minimized extends React.Component {
+	static displayName = 'DockedComposerMinimized';
+
+	state = {
+		isExpanded: false,
+		isOpen: true,
+	};
+
+	handleMinimize = (event, data) => {
+		this.setState({ isExpanded: false });
+	};
+
+	handleExpand = (event, data) => {
+		this.setState({ isExpanded: true });
+	};
+
+	handleClose = () => {
+		this.setState({ isOpen: false });
+	};
+
+	render() {
+		return this.state.isOpen ? (
+			<DockedComposer
+				header="What's New this Release?"
+				body={<Accordion />}
+				classNameContainer="hello-world"
+				assistiveText={{
+					expandButton: 'ExPaNd',
+					minimizeButton: 'MiNiMiZe',
+					closeButton: 'ClOsE',
+				}}
+				id="docked-composer"
+				events={{
+					onMinimize: this.handleMinimize,
```

Comment:
I haven't dove deep into this yet, but these look like on request pre-state change actions.
https://github.com/salesforce/design-system-react/blob/master/docs/codebase-overview.md#prefix-onrequestaction-to-pre-state-change-callbacks-functions